testsAT/src/test/resources/features/functionalAT/010_installation.feature

```diff
@@ -29,20 +29,27 @@ Feature: Installation testing with marathon-lb-sec
       | $.marathon-lb.vault_token                    | UPDATE  | !{vaultToken}                                                                                                                                                                                       | n/a     |
       | $.marathon-lb.instance_app_role              | UPDATE  | ${INSTANCE_APP_ROLE:-open}                                                                                                                                                                          | n/a     |
 
-    And I outbound copy 'target/test-classes/config.json' through a ssh connection to '/tmp'
-    When I run 'dcos package install --yes --app --options=/tmp/config.json ${PACKAGE:-marathon-lb-sec}' in the ssh connection
+  Scenario: [Install Marathon-lb][02] Install using config file and cli
+    #Copy DEPLOY JSON to DCOS-CLI
+    Given I open a ssh connection to '${DCOS_CLI_HOST:-dcos-cli.demo.stratio.com}' with user '${CLI_USER:-root}' and password '${CLI_PASSWORD:-stratio}'
+    When I outbound copy 'target/test-classes/config.${MARATHON_LB_VERSION:-0.3.1}.json' through a ssh connection to '/tmp/'
+    And I run 'dcos package install --yes --package-version=${MARATHON_LB_VERSION:-0.3.1} --options=/tmp/config.${MARATHON_LB_VERSION:-0.3.1}.json ${PACKAGE:-marathon-lb-sec}' in the ssh connection
```

Comment:
Change param name to  PACKAGE_MARATHON_LB (${PACKAGE:-marathon-lb-sec}) to avoid problems with other modules.